scripts/linters/js_ts_linter.py

```diff
@@ -845,6 +845,67 @@ def _check_constants_declaration(self):
 
         return summary_messages
 
+    def _check_comments(self):
+        """This function ensures that comments follow correct style."""
+        if self.verbose_mode_enabled:
+            python_utils.PRINT(
+                'Starting comment checks\n'
+                '----------------------------------------')
+        summary_messages = []
+        files_to_check = self.all_filepaths
+        allowed_terminating_punctuations = [
+            '.', '?', ';', ',', '{', '^', ')', '}']
+        excluded_phrases = [
+            '@ts-ignore', '--params', 'eslint-disable', 'eslint-enable',
+            'http://', 'https://']
+
+        failed = False
+        with linter_utils.redirect_stdout(sys.stdout):
+            for filepath in files_to_check:
+                file_content = FILE_CACHE.readlines(filepath)
+                file_length = len(file_content)
+                for line_num in python_utils.RANGE(file_length):
+                    line = file_content[line_num].strip()
+                    next_line = ''
+                    if line_num + 1 < file_length:
+                        next_line = file_content[line_num + 1].strip()
+
+                    # Check if comment contains any excluded phrase.
+                    word_is_present_in_excluded_phrases = any(
+                        word in line for word in excluded_phrases)
+
+                    # Comments may include a lowercase character at beginning
+                    # or may not use a punctuation at end if it contains a
+                    # excluded phrase e.g. "// eslint-disable".
+                    if word_is_present_in_excluded_phrases:
```

Comment:
We are missing one thing here. Comments like this:
`https://github.com/oppia/oppia/pull/9574/files%20is%20a%20useful%20link.` will be bypassed because it has `https` keyword in it which is added in `excluded_phrases`.

Though, most of the time link gets bigger in length so we've to shift into the next line while commenting.

What are your thoughts? @DubeySandeep

Reply:
Ping @DubeySandeep.

Could you please prioritise this? We are close to the milestone deadlines.